Description: Modify the sparql statement to make the keyword query a match that is included.
Fix support bug CESMB-107.

Author: Wenbo Guo <wenbo.guo@netgear.com>
---

--- samba-4.8.0.orig/source3/rpc_server/mdssvc/sparql_parser.y
+++ samba-4.8.0/source3/rpc_server/mdssvc/sparql_parser.y
@@ -373,7 +373,7 @@ static const char *map_expr(const char *
 		}
 		sparql = talloc_asprintf(talloc_tos(),
 					 "?obj %s ?%c "
-					 "FILTER(regex(?%c, '^%s$', 'i'))",
+					 "FILTER(regex(?%c, '^.*%s$', 'i'))",
 					 p->sparql_attr,
 					 s->var,
 					 s->var,
